test/Board/board.test.js

```diff
@@ -31,17 +31,17 @@ describe('Board', () => {
 
     describe('placeToken()', () => {
         test('will place a valid token in the board when a valid position is accepted', () => {
-            board.placeToken(new Token('X'), 2);
+            const token = new Token('X')
+            board.placeToken(token, 2);
```

Comment:
Ahh I'm going to say I "over-refactored". It was moreso an attempt to make the code look as "uniform" as possible where tokens were used in the tests. I definitely see why this could be a problem on a client base, however, and will be sure to keep the diffs in mind going forward! :)